telemetry/telemetry.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package telemetry
```

Comment:
0/5 but maybe we should give this analyser a more specific name, e.g. "config-telemetry" or something? Since that's all it does - it doesn't do anything else telemetry related.